examples/case_studies/bart_heteroscedasticity.ipynb

```diff
@@ -0,0 +1,586 @@
+{
```

Comment:
<p>One possibility is to manually select an explicit parametrization</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/519/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>Thanks! Fixed in https://github.com/pymc-devs/pymc-examples/pull/519/commits/a5b2acdf0ef1462da61531e575569adc045db660</p>